### ISSUE TITLE
Installation Path input field type

### ISSUE THREAD

**Issue by perosredo** [meefik/linuxdeploy#81]
Would be much easier if this was "browseable" instead of having to recall mount/path systems from memory and typing manually.

**Comment by NHellFire** [comment]
Yes, installation path needs to use a file picker instead of have to type it out.
